src/components/templates/VideoListPage.tsx

```diff
@@ -1,17 +1,48 @@
-import React from 'react'
-import { useParams } from 'react-router-dom'
+import React from "react";
+import { IVideoCard } from "../../common/interfaces/IVideoCard";
+import { VideoList } from "../organisms/videoList/VideoList";
+import { useQuery } from "../../common/utils/reactUtils";
+import { PageTitle } from "../atoms/pageTitle/PageTitle";
+import styled from "styled-components";
 
-function VideoListPage() {
-  const { role, genre } = useParams();
```

Comment:
`useParams` will return params defined in the route.  We don't have any since they're hypothetically coming in the query

Reply:
Oh, yeah, looking at your `useQuery` function this all makes complete sense.